spec/models/form/sales/pages/person_known_spec.rb

```diff
@@ -21,34 +21,9 @@
       expect(page.description).to be_nil
```

Comment:
Not a big deal but for consistency you could standardise which person index you use in the initial section in this (and equivalent) files. I.e. before the first `context "with person 2" do` block. For example above `page_id` has index 2 but `person_index` is 3. Tests still pass but perhaps a bit confusing?